include/drivers/uart.h

```diff
@@ -512,6 +512,12 @@ static inline int z_impl_uart_tx_abort(const struct device *dev)
  * immediately. After that event handler, set using @ref uart_callback_set,
  * is called with UART_RX_RDY or UART_RX_BUF_REQUEST events.
  *
+ * UART_RX_RDY event is generated in two cases:
+ * - if there are at least one byte received since previous UART_RX_RDY event
+ *   and no new bytes is received for the time longer than specified by the
+ *   @p timeout.
```

Comment:
```suggestion
 * - if there is at least one byte received since the previous UART_RX_RDY event
 *   and no new bytes are received for the time period specified by the
 *   @p timeout.
```

Reply:
thanks, applied.